sdks/java/extensions/avro/build.gradle

```diff
@@ -100,9 +100,9 @@ avroVersions.each { k, v ->
       }
 
       // only use compileClasspath on purpose to not include generated test files
-      // will recompile with the propper generated sources
+      // will recompile with the proper generated sources
       compileClasspath += sourceSets.main.output + configurations."avroVersion$k"
-      runtimeClasspath += configurations."avroVersion$k"
+      runtimeClasspath += sourceSets.main.output + configurations."avroVersion$k"
```

Comment:
It worked before possibly because core-construction-java previously depend on extensions/avro, so the main classes were in test runtime class path. But it is not the case after #30235